Pipfile

```diff
@@ -9,6 +9,14 @@ django-environ = "*"
 
 [dev-packages]
 flake8 = "*"
+pre-commit = "*"
+flake8-annotations = "*"
+flake8-import-order = "*"
 
 [requires]
 python_version = "3.9"
```

Comment:
A small suggestion
python 3.9 is relatively new
3.6+ would be better is what i feel

Reply:
right, let's actually go back to 3.8 for safe measures.